### ISSUE TITLE
Action links are regular links now

### ISSUE THREAD

**Issue by Gantron1** [idmillington/undum#50]
I tried to play your tutorial game on undum.com, and when I clicked the Action link, instead of performing the action, it tried to go to `http://undum.com/games/do-something?once` and gave a 404 error.

The also occurred when I downloaded the code and ran it both locally and on my remote webserver. It also occurred when I made my own game with an action link.

**Comment by idmillington** [comment]
I'm closing this because I can't replicate the problem. Feel free to reopen with more details if it's still bugging you.

**Comment by Gantron1** [comment]
I don't know how to reopen the issue, but it kind of is still bugging me. I got the same result just now in both Opera and Chrome. I select "What Undum Games are Made Of" from the choices menu, then "Let's move on again," and finally, "action."

It's `http://idmillington.github.io/undum/games/do-something?once` now that gives the 404 error, but the problem is exactly the same as two years ago.

I'm on Ubuntu 16.04, but that shouldn't make a difference, should it? Now I tried it in Firefox and luakit, got the same 404 error. Now, Chrome on an android phone--same 404 error.
